packages/uiweb/src/lib/components/chat/ChatProfile/ChatProfile.tsx

```diff
@@ -56,7 +56,7 @@ export interface IChatProfileUserInfo {
 // Exported Functions
 export const ChatProfile: React.FC<IChatProfile> = ({
   chatId,
-  closeModalOnClickAway,
+  closeGroupInfoModalOnClickAway,
```

Comment:
rename closeGroupInfoModalOnClickAway to closeChatProfileInfoModalOnClickAway everywhere